lambdaclass/mirra_backend
# [GH-971-969] - Increase wall colliders and fix items not spawning inside the square

## Motivation

Having an square walls, is causing some troubles with things that could appear beyond these walls


Closes #971
Closes #969 

## Summary of changes

- Increased map radius size
- Increased external walls size (almost duplicataed)
- Added an square wall field to map configurations

## How to test it?

- Play the game and check that the items spawns inside the square (you'll have to print the random position created in `random_position_in_square/5`  and check that they don't go over the walls)
- Check that valtimer is not able to go trough the external walls

## Checklist
